src/vacuum_map_parser_ijai/map_data_parser.py

```diff
@@ -104,6 +111,29 @@ def _parse_image(self) -> tuple[ImageData, dict[int, Room], set[int]]:
         image_height = self.robot_map.mapHead.sizeY
         image_size = image_height * image_width
         _LOGGER.debug("width: %d, height: %d", image_width, image_height)
+
+        mapData_temp = self.robot_map.mapData.mapData
+
+        if (len(set(mapData_temp).symmetric_difference([0, 128, 127])) == 0 and len(self.robot_map.roomChain) > 0 and self.robot_map.mapType == 0):
+            buautify_obj = Beautify.BeautifyMap()
+            buautify_obj.setMap(self.robot_map)
+            buautify_obj.transform()
+            buautify_obj.roomColorByChain(self.robot_map.roomChain)
+            buautify_obj.fillInternalObstacles()
+
+            mapData_temp = buautify_obj.getMap()
+
+            for i in range(len(mapData_temp)):
+                if mapData_temp[i] < 0:
+                    mapData_temp[i] = (256 + mapData_temp[i]) % 256
+                elif mapData_temp[i] > 255:
```

Comment:
Please, consider naming 'magic' constants

Reply:
This is a method for normalizing mapData to bytes, now moved into BeautifyMap class